ethcore/res/ethereum/ewc.json

```diff
@@ -883,13 +888,24 @@
 					"0": {
 						"price": { "alt_bn128_pairing": { "base": 100000, "pair": 80000 }}
 					},
-					"0x7fffffffffffff": {
-						"info": "EIP 1108 transition",
+					"0x459bbf": {
+						"info": "EIP 1108 transition at block 4_561_855 (0x459bbf)",
 						"price": { "alt_bn128_pairing": { "base": 45000, "pair": 34000 }}
 					}
 				}
 			}
 		},
+		"0x0000000000000000000000000000000000000009": {
+			"builtin": {
```

Comment:
one question though, why has the other builtins `balance = 1`?

Reply:
@ngyam ^^^?

Reply:
good observation, not sure if I can give a sound answer to that because I didn't do that part back then. Maybe to be on the safe side of the init of those accounts (if it even makes sense).
As far as I know it's not necessary. @niklasad1

Reply:
Ok, @dvdplm explained it well: https://github.com/paritytech/parity-ethereum/pull/11108#issuecomment-537138835

Note, that changing `balance` will likely end up with a different state root but I just wanted to highlight this because we had a few bugs related this in the past.

We really should document this somewhere but I guess it is not documented because JSON doesn't support comments.

Reply:
thanks for the clarification. So in theory all should be good to go without adding 1 wei to this account balance, right?

Reply:
I don't want to change the chainspec if not necessary

Reply:
but if there is a slight change that otherwise there is a risk of some bugs, then of course it makes sense

Reply:
update: we decided to add this "missing" 1 wei just to be safe. I update the PR tomorrow